packages/components/src/components/modal/modal.tsx

```diff
@@ -279,6 +283,7 @@ const Modal = ({
             onUnmount={onUnmount}
             portalId={portalId}
             renderTitle={renderTitle}
+            should_close_on_click_outside={should_close_on_click_outside}
```

Comment:
If in `<Modal>` component `should_close_on_click_outside` has a default value (=== `false`), then why we need to add same default value in `<ModalElement>`? It will be not optional there.